project.clj

```diff
@@ -1,70 +1,43 @@
-(def tk-version "1.4.0")
-(def ks-version "1.3.0")
-
-
-(defproject puppetlabs/trapperkeeper-status "0.6.1-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-status "0.7.0-SNAPSHOT"
   :description "A trapperkeeper service for getting the status of other trapperkeeper services."
   :url "https://github.com/puppetlabs/trapperkeeper-status"
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0"}
 
-  :pedantic? :abort
-
-  :exclusions [org.clojure/clojure]
+  :min-lein-version "2.7.1"
 
-  :dependencies [[org.clojure/clojure "1.7.0"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.2.5"]
+                   :inherit [:managed-dependencies]}
 
-                 ;; Dependencies which resolve version conflicts via
-                 ;; :pedantic? :abort in transitive dependencies
-                 [clj-time "0.11.0"]
-                 [ring "1.4.0"]
-                 [commons-codec "1.10"]
-                 [org.clojure/tools.macro "0.1.5"]
-                 [org.clojure/tools.logging "0.3.1"]
```

Comment:
Should we still have a managed dep listed for this since we have some code which has a `require` for it?